wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/ProfileDeviceDetailsView.swift

```diff
@@ -85,13 +85,13 @@ struct ProfileDeviceDetailsView: View {
         HStack {
             Text(L10n.Localizable.Profile.Devices.Detail.ShowMyDevice.title)
                 .padding(.all, ViewConstants.Padding.standard)
-                .foregroundColor(SemanticColors.Label.textDefault.swiftUIColor)
```

Comment:
Maybe I didn't notice, but did you also remove swiftUIColor property from the UIColor extension?

Reply:
Yes, it doesn't make sense, we can just call `Color(:uiColor)` directly.
```
var swiftUIColor: Color {
    return Color(uiColor: self)
}
```